src/tools/interopserver/InteropServer.cpp

```diff
@@ -187,7 +187,8 @@ HttpRequest::Process()
     }
 
     if (strcmp("/", PathStart) == 0) {
-        PathStart = "/index.html";
+        char index[] = "/index.html";
+        PathStart = index;
```

Comment:
`PathStart` outlives the scope of `index`. Isn't that a bug?

Reply:
I don't think the memory is released from the stack before the end of the function. But I can move it outside, to make it clearer.